numpy/random/mtrand.pyx

```diff
@@ -643,7 +642,7 @@ cdef class RandomState:
 
     def randint(self, low, high=None, size=None, dtype=int):
         """
-        randint(low, high=None, size=None, dtype='l')
+        randint(low, high=None, size=None, dtype=int)
```

Comment:
```suggestion
        randint(low, high=None, size=None, dtype=np.int_)
```
As written this is a behavior change

Reply:
Oh wait, this was actually correct before. Sorry, I confused the docstring with the function def.